src/main/java/com/Project/accountManager/entities/Account.java

```diff
@@ -0,0 +1,31 @@
+package com.Project.accountManager.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.Data;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+@Entity
+@Table(name="account")
```

Comment:
Same like here https://github.com/serhatbsts/accountManager/pull/2/files#r1612716592

Reply:
OK, necessary corrections will be made.